.ci_scripts/run_build.sh

```diff
@@ -20,6 +20,10 @@ case "$BUILD_TARGET" in
 	echo "Creating disk image"
 	hdiutil create -volname Julius -srcfolder julius.app -ov -format UDZO julius.dmg
 	;;
+"flatpak")
+	flatpak-builder repo com.github.bvschaik.julius.json --install-deps-from=flathub --keep-build-dirs
+	cp .flatpak-builder/build/julius/res/version.txt res/version.txt
```

Comment:
Why do we need this copy action? `res/version.txt` should be generated by CMake -- I'm guessing that flatpack-builder uses cmake under the hood?

Reply:
Yes it is using cmake under the hood but it is building it out of tree (in `./.flatpak-builder/build/julius`) and thus isn't being found afterwards by the `build_upload.sh` script